src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,28 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
-
+        /// <summary>
+        /// Switch the order of how two elements are geometry is joined. 
+        /// Note that changing this in 3D will also affect your 2D views.
```

Comment:
```suggestion
        /// Note that changing this will affect both 3D and 2D views and that it is not the recommended way of managing wall joins.
```